content/webapp/pages/concepts/[conceptId].tsx

```diff
@@ -366,8 +463,41 @@ export const ConceptPage: NextPage<Props> = ({
 
   const { frequentCollaborators, relatedTopics } =
     conceptResponse.relatedConcepts || {};
+  const relatedConceptsGroupLabel = 'Related topics';
+  const personOrAllFields =
+    conceptResponse.type === 'Person' || themePagesAllFields;
+  const navLinks = [
+    sectionsData.by.images?.totalResults
+      ? {
+          text: `Images ${getThemeTabLabel('by', conceptResponse.type)}`,
+          url: `#images-${getThemeTabLabel('by', conceptResponse.type)}`,
+        }
+      : undefined,
+    sectionsData.about.images?.totalResults
+      ? {
+          text: `Images ${getThemeTabLabel('about', conceptResponse.type)}`,
+          url: `#images-${getThemeTabLabel('about', conceptResponse.type)}`,
+        }
+      : undefined,
+    sectionsData.in.images?.totalResults
+      ? {
+          text: `Images ${getThemeTabLabel('in', conceptResponse.type)}`,
+          url: `#images-${getThemeTabLabel('in', conceptResponse.type)}`,
+        }
```

Comment:
Maybe one more for the On this page ticket, but thinking a helper might be nice here, for readability purposes. It's quite stacked!

Reply:
Is [this](https://github.com/wellcomecollection/wellcomecollection.org/pull/12063/commits/d7850b12f060b332656e44ed48bc3bd18cf36fd4) better? I'm not sure

Reply:
I see what you mean but I do think it's more readable. I could be due to the spacing and the added comments, and not needing to worry about what `sectionsData.in.images` stands for.